src/api/handlers/channel.ts

```diff
@@ -301,6 +302,38 @@ export async function createInvite(
   return RequestManager.post(endpoints.CHANNEL_INVITES(channelID), options);
 }
 
+/** Returns an invite for the given code. */
+export function getInvite(inviteCode: string) {
+  //TODO(itohatweb): types: Better return type
+  return RequestManager.get(endpoints.INVITE(inviteCode));
+}
+
+/** Deletes an invite for the given code. Requires `MANAGE_CHANNELS` or `MANAGE_GUILD` permission */
+export async function deleteInvite(
+  channelID: string,
+  inviteCode: string,
+) {
+  const hasPerm = await botHasChannelPermissions(channelID, [
+    "MANAGE_CHANNELS",
+  ]);
+
+  if (!hasPerm) {
+    const channel = await cacheHandlers.get("channels", channelID);
+    if (!channel) throw new Error(Errors.CHANNEL_NOT_FOUND);
+
+    const hasManageGuildPerm = await botHasPermission(channel.guildID, [
+      "MANAGE_GUILD",
+    ]);
+
+    if (!hasManageGuildPerm) {
+      throw new Error("MISSING_MANAGE_CHANNELS OR MANAGE_GUILD");
```

Comment:
This should either use the Errors enum or we should redo all errors. The enum allows easy use of error strings to provide an easily 18next translation response to the end user.

Reply:
Gonna just throw `MISSING_MANAGE_CHANNELS`

Reply:
That would be incorrect then.

Reply:
It is and it is not. MISSING_MANAGE_CHANNELS or MANAGE_GUILD is required but I cannot throw 2 errors at the same time.